packages/discovery-react-components/src/components/DocumentPreview/__tests__/DocumentPreview.test.tsx

```diff
@@ -1,118 +1,341 @@
 import React from 'react';
-import { act, render } from '@testing-library/react';
+import { act, render, BoundFunction, GetByText, FindAllBy } from '@testing-library/react';
 import omit from 'lodash/omit';
 import { NoAuthAuthenticator } from 'ibm-watson/auth';
 import DiscoveryV2 from 'ibm-watson/discovery/v2';
 import DiscoverySearch from 'components/DiscoverySearch/DiscoverySearch';
 import DocumentPreview from '../DocumentPreview';
-import { document as doc } from '../__fixtures__/Art Effects.pdf';
-import docJson from '../__fixtures__/Art Effects Koya Creative Base TSA 2008.pdf.json';
+import pdfDocument from '../__fixtures__/Art Effects Koya Creative Base TSA 2008.pdf.json';
 import passages from '../__fixtures__/passages';
+import 'utils/test/createRange.mock';
+import { SearchApiIFC, SearchContextIFC } from 'components/DiscoverySearch/DiscoverySearch';
+import { wrapWithContext } from 'utils/testingUtils';
+
+function isValidHighlight(
```

Comment:
Not necessary for this PR, but take a look at custom jest expect functions. This could be changed to `toBeValidHighlight`, for example. We do it in one of our other tests already.

Reply:
Went ahead and made this change!